support/cas-server-support-oauth-core-api/src/main/java/org/apereo/cas/support/oauth/web/endpoints/OAuth20UserProfileEndpointController.java

```diff
@@ -141,6 +141,6 @@ protected String getAccessTokenFromRequest(final HttpServletRequest request) {
             }
         }
         LOGGER.debug("[{}]: [{}]", OAuth20Constants.ACCESS_TOKEN, accessToken);
-        return accessToken;
+        return OAuth20Utils.getAccessTokenId(accessToken, getOAuthConfigurationContext().getOAuthAccessTokenIdExtractor());
```

Comment:
Same as above. Simply call the extractor, rather than pushing the functionality down the utils.